### PR TITLE
Dvpackager

### PR DESCRIPTION
**Pull request by savcampbell**
adds script for packaging dv files without creating prores file

### CODE REVIEW

File: dvpackager [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+while [ "${*}" != "" ] ; do
+    input="$1"
+    makelossless -j "${input}" && makeprores "${input}" && makeaccess "${input}" && makeframes "${input}" && makemetadata "${input}" && checksumpackage "${input}"
```
Comment (dericed) [inline dvpackager]: wasn't it intended to remove makeprores in this case?

Comment (savcampbell) [reply dvpackager]: Yep. Copied the wrong file.

**Comment by savcampbell** [comment]
Should be ok now?

**Comment by dericed** [comment]
Looks good, I recommend using "Rebase and merge" when merging so this is all squashed into a single commit. Also note that you'll have to add this to https://github.com/DanceHeritageCoalition/homebrew-dhc/blob/master/dpdpscripts.rb.

**Comment by savcampbell** [comment]
Sure. Thank you. It has clearly been a while since I've used Github :)

**Pull request merged**